docker/.env

```
@@ -209,6 +218,10 @@ UV_CUBE_AGENT_SERVER_CERT=
 UV_CUBE_AGENT_SERVER_KEY=
 UV_CUBE_AGENT_INSTANCE_ID=
 
+# Agent Client Certificates for mTLS
+UV_CUBE_AGENT_CLIENT_CERT=/etc/cube/certs/client.crt
+UV_CUBE_AGENT_CLIENT_KEY=/etc/cube/certs/client.key
+UV_CUBE_AGENT_CLIENT_CA=/etc/cube/certs/ca.pem
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UV_CUBE_AGENT_CLIENT_CA key should go before the UV_CUBE_AGENT_CLIENT_CERT key

(UnorderedKey)